service/dynamodb/dynamodbattribute/tag.go

```diff
@@ -33,7 +33,7 @@ func (t *tag) parseJSONTag(structTag reflect.StructTag) {
 }
 
 func (t *tag) parseTagStr(tagStr string) {
-	parts := strings.SplitN(tagStr, ",", 2)
+	parts := strings.SplitN(tagStr, ",", 3)
```

Comment:
I think we can just change this to `Split` instead of `SplitN`. I don't think we need to limit the number of parts in the tag.